libevmasm/SemanticInformation.cpp

```diff
@@ -188,3 +188,56 @@ bool SemanticInformation::invalidatesStorage(Instruction _instruction)
 		return false;
 	}
 }
+
+bool SemanticInformation::invalidInPureFunctions(Instruction _instruction)
```

Comment:
I'd call this "readsFromState".

Reply:
Sorry, I think it should go back to your original one as the other one cannot be renamed it is better to have it consistently.